website/docs/d/launch_configuration.html.markdown

```diff
@@ -41,6 +41,7 @@ This data source exports the following attributes in addition to the arguments a
     * `http_put_response_hop_limit` - The desired HTTP PUT response hop limit for instance metadata requests.
 * `security_groups` - List of associated Security Group IDS.
 * `associate_public_ip_address` - Whether a Public IP address is associated with the instance.
+* `primary_ipv6` - Whether the first IPv6 GUA will be made the primary IPv6 address.
```

Comment:
This updates the wrong docs. :/  Launch Configurations don't support this attribute, and are deprecated. I think this was meant to be added to `aws_launch_template` docs